doc/nrf/config_and_build/configuring_app/advanced_building.rst

```diff
@@ -0,0 +1,109 @@
+  For example, :file:`child_image/mcuboot/prj_release.conf` can be used to define ``release`` build type for ``mcuboot`` child image.
+
+.. build_types_overview_end
+
+The Devicetree configuration is not affected by the build type.
+
+.. note::
+    For an example of an application that is using build types, see the :ref:`nrf_desktop` application (:ref:`nrf_desktop_requirements_build_types`) or the :ref:`nrf_machine_learning_app` application (:ref:`nrf_machine_learning_app_requirements_build_types`).
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
+
+      .. build_types_selection_vsc_start
+
+      To select the build type in the |nRFVSC|:
+
+      1. When `building an application <How to build an application_>`_ as described in the |nRFVSC| documentation, follow the steps for setting up the build configuration.
+      #. In the **Add Build Configuration** screen, select the desired :file:`.conf` file from the :guilabel:`Configuration` drop-down menu.
+      #. Fill in other configuration options, if applicable, and click :guilabel:`Build Configuration`.
+
+      .. build_types_selection_vsc_end
+
+   .. group-tab:: Command line
+
+      .. build_types_selection_cmd_start
+
+      To select the build type when building the application from command line, specify the build type by adding the following parameter to the ``west build`` command:
+
+      .. parsed-literal::
+         :class: highlight
+
+         -- -DCONF_FILE=prj_\ *selected_build_type*\.conf
+
+      For example, you can replace the *selected_build_type* variable to build the ``release`` firmware for ``nrf52840dk_nrf52840`` by running the following command in the project directory:
+
+      .. parsed-literal::
+         :class: highlight
+
+         west build -b nrf52840dk_nrf52840 -d build_nrf52840dk_nrf52840 -- -DCONF_FILE=prj_release.conf
+
+      The ``build_nrf52840dk_nrf52840`` parameter specifies the output directory for the build files.
+
+      .. build_types_selection_cmd_end
+
+If the selected board does not support the selected build type, the build is interrupted.
+For example, for the :ref:`nrf_machine_learning_app` application, if the ``nus`` build type is not supported by the selected board, the following notification appears:
+
+.. code-block:: console
+
+   Configuration file for build type ``nus`` is missing.
+
+Optional build parameters
+*************************
+
+Here are some of the possible options you can use:
+
+* Some applications contain configuration overlay files that enable specific features.
+  These can be added to the ``west build`` command as follows:
+
+  .. parsed-literal::
+     :class: highlight
+
+     west build -b *build_target* -- -DOVERLAY_CONFIG="overlay-feature1.conf;overlay-feature2.conf"
+
+  See :ref:`configuration_permanent_change` and Zephyr's :ref:`zephyr:west-building-cmake-args` for more information.
+* You can include the *directory_name* parameter to build from a directory other than the application directory.
```

Comment:
```suggestion
* You can include the *directory_name* parameter to build from a directory other than the current directory.
```
Does this mean the source directory? If so, let's call it like that and provide an example. Source directory must be put before `--` so it may not be easy to understand without an example.